CoreFoundation/URL.subproj/CFURLSessionInterface.h

```diff
@@ -48,6 +48,10 @@ typedef struct CFURLSessionEasyCode {
     int value;
 } CFURLSessionEasyCode;
 
+CF_EXPORT const char * _Nonnull CFURLSessionErrorDescription(int value);
```

Comment:
CF API really should use a `CFStringRef` as a result instead of C strings. That resolves the memory management question of the result for callers.

Reply:
Changed to CFString. Unsure of the allocator I should be using if you want to check it.

Reply:
CF (almost) always uses `kCFAllocatorSystemDefault` internally.